### DISCUSSION TITLE
Access docker containers

### DISCUSSION THREAD

**Discussion by tutods** [abiosoft/colima#421] [Q&A]
Hello everyone. Is my first time with colima, and I have problems access the containers using localhost:<port>. Anyone can help me?

**Comment by abiosoft** [comment]
Can you specify the commands used for running the containers?

**Comment by cyberwombat** [comment]
Similar issue - I use: `docker run --name dynamodb -d -p 8000:8000 amazon/dynamodb-local` - only accessible on 127.0.0.1 now (worked on Docker Desktop)

**Comment by tutods** [comment]
Now I'm facing a different problem.
I start Colima and docker it's running ok, but after a while, if I do a simple command, like `docker ps` I have a docker daemon error.
Already followed the FAQ instructions, but it did not work.

**Comment by abiosoft** [comment]
@tutods you are likely on v0.5.0. The issue is specific to the new macOS virtualization driver, which has been removed as default from v0.5.1. 

It is recommended to stick to QEMU for now.
